integration-tests/src/integrationTest/java/pl/allegro/tech/hermes/integrationtests/management/TopicManagementTest.java

```diff
@@ -765,6 +765,22 @@ public void shouldNotAllowNonAdminUserToEnableChaos() {
                 .contains("User is not allowed to update chaos policy for this topic");
     }
 
+    @Test
+    public void shouldNotAllowNonAdminUserToEnableChaos2() {
+        // given
+        Topic topic = hermes.initHelper().createTopic(topicWithRandomName().withFallbackToRemoteDatacenterEnabled().build());
+        TestSecurityProvider.setUserIsAdmin(false);
+        PatchData patchData = PatchData.from(ImmutableMap.of("description", "a"));
+
+        // when
+        WebTestClient.ResponseSpec response = hermes.api().updateTopic(topic.getQualifiedName(), patchData);
+
+        //then
+        response.expectStatus().isOk();
+        Topic t = hermes.api().getTopicResponse(topic.getQualifiedName()).returnResult(Topic.class).getResponseBody().blockFirst();
+        System.out.println("Aaa");
```

Comment:
I think this was for debugging purposes 😛

Reply:
removed